app/utils/nextpage/employer/IsBusinessRegisteredForPAYENextPage.scala

```diff
@@ -39,4 +41,32 @@ trait IsBusinessRegisteredForPAYENextPage {
     }
   }
 
+  implicit val isBusinessRegisteredForPAYEEI
```

Comment:
naming is different from below val, naming conventions should be the same. Naming should be `eiIsBusinessRegisteredForPAYE` as per other files and methods